test/serializers/default_include_test.rb

```diff
@@ -0,0 +1,52 @@
+require 'test_helper'
+
+module ActiveModel
+  class Serializer
```

Comment:
Ok, thanks @NullVoxPopuli.
My initial idea was to have a cascade-include behaviour, but I can see why you prefer going down this route. I'll probably need to move my code then because AFAIK `associations` method is called for each serialiser, even nested ones.
It would make more sense to have the merge happening when the `ActiveModelSerializers::SerializableResource` is created for rendering